state/mocks/store.go

```diff
@@ -3,12 +3,13 @@
 package mocks
 
 import (
-	types "github.com/tendermint/tendermint/proto/tendermint/types"
+	ostraconstate "github.com/line/ostracon/proto/ostracon/state"
```

Comment:
How about using `ocstate`?

Reply:
This is a auto-generated file by mockery.